src/components/common/buttons/CustomButton.js

```diff
@@ -167,7 +165,7 @@ const CustomButton = (props: ButtonProps) => {
       theme={{ ...theme, roundness }}
       uppercase={uppercase}
       disabled={disabled || loading}
-      onPress={_onPress}
+      onPress={onPress}
```

Comment:
@serdiukov-o-nordwhale can you explain where and how to use useOnPress?

Reply:
@sirpy @serdiukov-o-nordwhale the removal on this file is an extension of the problem described in the other comment

Reply:
> @serdiukov-o-nordwhale can you explain where and how to use useOnPress?

@sirpy https://github.com/GoodDollar/GoodDAPP/issues/2624#issuecomment-712073881

Reply:
fixed. per useOnPress rules and my conversation with @serdiukov-o-nordwhale I removed all unnecessary useOnPress on top components and only kept it for the bottom component. I also had to make that component functional as it was working like a class